Cargo.toml

```diff
@@ -14,3 +14,5 @@ debug = 2 # full debug info
 
 [profile.release]
 debug = 1 # line tables only
+lto = true
+codegen-units = 1
```

Comment:
Is this something we want to put in the cargo config, or could we not make sure to have it as a command line argument in the CI ? 
I'm thinking we dan't want to slow all of our build times. Only the build time to generate the github release.
WDYT ?